terraform/main.tf

```diff
@@ -40,6 +40,7 @@ module "gcs_buckets" {
   version    = "~> 3.2"
   prefix     = ""
   project_id = var.project_id
+  location   = var.vertex_region # Added by JAN - This is missing my sandbox in comparison to the mlops playgrounds pipeline runner SA. I am hoping it will solve the issue with the model training job restarting because of error: https://console.cloud.google.com/logs/query;cursorTimestamp=2022-09-07T08:51:07.281723787Z;pinnedLogId=2022-09-07T08:40:56.206185934Z%2F1fweneyc9ca;query=severity%3DERROR%0Atimestamp%3D%222022-09-07T08:51:07.281723787Z%22%0AinsertId%3D%22pt460hc3cb%22;timeRange=2022-09-07T08:04:58.023Z%2F2022-09-07T11:04:58.023596Z?project=dt-jan-sandbox-dev
```

Comment:
Before finalising this PR, remove reference to your sandbox project

Reply:
👍